infra/terraform/modules/service/batch.tf

```
@@ -110,6 +118,16 @@ module "batch" {
   job_definitions = local.jobs
 }
 
+module "eventbridge" {
+  source  = "terraform-aws-modules/eventbridge/aws"
+  version = "~> 3.7.1"
```

```suggestion
  version = "~> 3.7"
```

Could we loosen this a little to allow patch versions and be consistent with other version constraints?